### PR TITLE
Recording webhooks

### PR DESCRIPTION
**Pull request by frostbyte73**

### CODE REVIEW

File: pkg/service/recordingservice.go [hunk 1]
```diff
@@ -87,3 +98,57 @@ func (s *RecordingService) EndRecording(ctx context.Context, req *livekit.EndRec
 
 	return &livekit.RecordingResponse{RecordingId: req.RecordingId}, nil
 }
+
+func (s *RecordingService) resultsWorker() {
+	sub, err := s.mb.Subscribe(context.Background(), utils.RecordingResultChannel)
```
Comment (davidzhao) [inline pkg/service/recordingservice.go:103]: should sub be closed?

**Review by davidzhao — APPROVED** [review]
lg, just one comment

File: pkg/service/recordingservice.go (hunk 1 above)
Comment (frostbyte73) [reply pkg/service/recordingservice.go:103]: it stays subscribed

Comment (frostbyte73) [reply pkg/service/recordingservice.go:105]: it never exits

Comment (frostbyte73) [reply pkg/service/recordingservice.go:105]: I can add in a Stop

**Pull request merged**